README.md

```diff
@@ -26,13 +23,13 @@ A simple setup using:
 git clone https://github.com/kensonjohnson/relativepath.tech.git
 ```
 
-2. `cd` into the `relativepath.tech` directory:
+2. Navigate to the `relativepath.tech` directory:
 
 ```bash
 cd ./relativepath.tech
 ```
 
-3. Run the `setup.sh` bash script to make sure your system has necessary requirements:
+3. Run the `setup.sh` bash script to install requirements:
 
 ```bash
 bash setup.sh
```

Comment:
What if we rename this file to local.sh, then later if we have like local-darwin.sh or local-powershell.sh things will make sense? I think naming the script based on the context it runs or the context+job it does is maybe something that could work.

Reply:
> What if we rename this file to local.sh, then later if we have like local-darwin.sh or local-powershell.sh things will make sense? I think naming the script based on the context it runs or the context+job it does is maybe something that could work.

How about passing a flag or argument like `setup.sh --macos`?

Reply:
In my experience it's easier to have a script per platform than a complex script that tries to do each platform.